### DISCUSSION TITLE
配合magic-signer-guide尝试登录时出现错误：Recursive Update

### DISCUSSION THREAD

**Discussion by Lzhjk1** [cssxsh/fix-protocol-version#87] [General]
使用MCL加载的
MCL日志：
[mcllog.txt](https://github.com/cssxsh/fix-protocol-version/files/12911492/mcllog.txt)
magic-signer-guide的docker镜像日志：
[docker_magic-signer-guide_log.txt](https://github.com/cssxsh/fix-protocol-version/files/12911498/docker_magic-signer-guide_log.txt)
docker日志的时间出问题了，像是时区不对，早8个小时，尝试修正没弄好，但是进容器输入date命令显示的时间是正常的，不知道这个问题有没有影响

192.168.5.218是我局域网另外一台电脑，上面有docker我就直接在那上面运行了，没有在本地运行

**Comment by cssxsh** [comment]
销毁会话的时候出了一些问题
